### PR TITLE
Add SessionContents search() test

### LINKED ISSUE #746: Add SessionContents search() test
## Kind (Required)

- Other

 ## Overview (Required)

- You can test it like FiltersTest

 ## Links	

- https://github.com/DroidKaigi/conference-app-2019/blob/master/model/src/jvmTest/kotlin/io/github/droidkaigi/confsched2019/model/FiltersTest.kt#L9

### PR DESCRIPTION
**Pull request by xnxaxo**
## Issue
- close #746

## Overview (Required)
- Add test for SessionContents
- Refer to SessionContentsTest

## Links
-

### CODE REVIEW

**Comment by takahirom** [comment]
Please fix danger warnings 🙏

File: feature/session/src/test/java/io/github/droidkaigi/confsched2019/SessionDummyDatas.kt [hunk 1]
```diff
@@ -80,3 +81,77 @@ fun firstDummySpeechSession(): SpeechSession {
...
+        room = Room(
+            id = 22,
+            name = "Room2"
+        ),
+        format = "",
+        lang = Lang.JA,
+        category = Category(
+            id = 22,
+            name = LocaledString(
+                ja = "猫アーキテクチャー",
```
Comment (takahirom) [inline feature/session/src/test/java/io/github/droidkaigi/confsched2019/SessionDummyDatas.kt]: 🐈

File: feature/session/src/test/java/io/github/droidkaigi/confsched2019/model/SessionContentsTest.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import io.github.droidkaigi.confsched2019.dummySpeakerData
+import io.github.droidkaigi.confsched2019.dummySpeechSessionData
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class SessionContentsTest {
+
+    // Session TestCase
+
+    @Test fun search_WhenSpeechSessionFilteredAndTitleEn() =
```
Comment (takahirom) [inline feature/session/src/test/java/io/github/droidkaigi/confsched2019/model/SessionContentsTest.kt]: Can you do it with `Parameterized`? It is a little difficult (I can't understand this test without 2 definition dump 😭 )
https://github.com/DroidKaigi/conference-app-2019/blob/2c9030940154f367d36d10f0ecd6911abe4879d3/model/src/jvmTest/kotlin/io/github/droidkaigi/confsched2019/model/FiltersTest.kt#L25

Comment (xnxaxo) [reply feature/session/src/test/java/io/github/droidkaigi/confsched2019/model/SessionContentsTest.kt]: Let's try a test case using Parameterized.

**Comment by xnxaxo** [comment]
@takahirom 
Parameterized used test case.


sorry.
Since I could not write an appropriate title for each testcase in English, I added what I am targeting.

**Pull request merged**